source/sdk/lang/Memory.ooc

```diff
@@ -1,53 +1,20 @@
-
-// as goofy as it sounds, memory-related routines are actually in string.h
-// yay C. (Also, Google Native Client doesn't have memory.h)
```

Comment:
I think this comment should stay, or a more serious one should replace it. It's far from obvious to the next person who looks at this file that we need `string` for anything.